docs/source/markdown/podman-manifest-push.1.md

```diff
@@ -32,6 +32,10 @@ environment variable. `export REGISTRY_AUTH_FILE=path`
 Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry. (Default: /etc/containers/certs.d)
 Please refer to containers-certs.d(5) for details. (This option is not available with the remote Podman client, including Mac and Windows (excluding WSL2) machines)
 
+#### **--compression-format**=**gzip** | *zstd* | *zstd:chunked*
+
+Specifies the compression format to use.  Supported values are: `gzip`, `zstd` and `zstd:chunked`.  The default is `gzip`.
```

Comment:
Sinze the default comes from containers.conf you should mention it.

Reply:
I have added a mention to it at the end of the sentence (and also updated `docs/source/markdown/podman-push.1.md` accordingly since I copied it from these).